### ISSUE TITLE
thread 'tokio-runtime-worker' panicked at 'attempt to subtract with overflow'

### ISSUE THREAD

**Issue by raptpz** [harlanc/xiu#17]
I've configured xiu with a simple RTMP server:
[rtmp]
enabled = true
port = 1935

I then publish a sample RTMP stream with ffmpeg:
ffmpeg -re -stream_loop -1 -i sample_video.mp4 -vcodec libx264 -acodec aac -f flv rtmp://127.0.0.1:1935/live/test 

When the input stream is stopped, I got the following error on xiu server:

thread 'tokio-runtime-worker' panicked at 'attempt to subtract with overflow', C:\Users\Administrator\.cargo\registry\src\github.com-1ecc6299db9ec823\rtmp-0.0.1
0\src\chunk\unpacketizer.rs:422:25

Is there a way to avoid it ?  
 
Notes:
- tried with xiu 0.0.9
- both "xiu" and "ffmpeg" are running on Windows Server O.S.

**Comment by harlanc** [comment]
Thanks for your report, can the problem be reproduced stably or it happens randomly?

**Comment by raptpz** [comment]
Thank you for your reply, it can be reproduced stably.
Let me know if you needed further details on the issue, or if you can reproduce too

**Comment by harlanc** [comment]
@raptpz  I did some fix , try the latest xiu version v0.0.10.
Thanks.

**Comment by raptpz** [comment]
I tried v0.0.10 and I confirm that now the behaviour is correct: the stream session closure is correctly handled 
Thank you very moch,
best regards

**Comment by raptpz** [comment]
Today, after several hours of "live" streaming in the same conditions (ffmpeg as source of streaming), when I stopped the ffmpeg input stream, I got a similar panich error in "xiu v0.0.10" executable:
"**thread 'tokio-runtime-worker' panicked at 'attempt to add with overflow'**"
(note: this time the error was with "add" instead of "subtract")

I will try to reproduce it, since it seems to happen only after several hours of streaming: with few minutes of streaming no more issue,
Do you want to open a new issue or keep this re-opened?

**Comment by harlanc** [comment]
Just keep this issue re-opened,do you record the detailed stack info about the add overflow？Thanks.

**Comment by raptpz** [comment]
I will keep it running until I get the issue again and then publish here
Thank you

**Comment by harlanc** [comment]
just close this issue since the problem cannot be reproduced...
